sysdig/resource_sysdig_secure_cloud_auth_account.go

```diff
@@ -152,7 +153,7 @@ func resourceSysdigSecureCloudauthAccount() *schema.Resource {
 				Elem:     accountFeatures,
 			},
 			SchemaComponent: {
-				Type:     schema.TypeList,
+				Type:     schema.TypeSet,
```

Comment:
This change appears to remove the problem where component ordering surfaced spurious diff errors.
cc: @ravinadhruve10 @wideawakening